webapp/src/components/admin_settings/jaas_section.tsx

```diff
@@ -0,0 +1,118 @@
+import * as React from 'react';
+
+type Props = {
+    disabled: boolean,
+    onAppIDChange: any,
+    onApiKeyChange: any,
+    onPrivateKeyChange: any,
+    onEmbeddedChange: any,
+    apiKey: string,
+    appID: string,
+    privateKey: string
+    embedded: boolean
+};
+
+export default class JaaSSection extends React.Component<Props> {
+    constructor(props: any) {
+        super(props);
+        console.log('JaaSSection');
+    }
+
+    render() {
+        // TODO replace with common components
```

Comment:
I guess this TODO is for the future, not for now, right?

Reply:
Yes, just made something to work, improvements and features might be added later.

Reply:
Removed the TODO.